.circleci/config.yml

```diff
@@ -49,12 +49,18 @@ jobs:
       - image: circleci/python:3.5
         environment:
           TOXENV: doctest
-  lint:
+  lint-py36:
     <<: *common
     docker:
       - image: circleci/python:3.6
         environment:
-          TOXENV: lint
+          TOXENV: lint-py36
+  lint-py37:
```

Comment:
I don't think it adds a ton of value to run the linter in multiple pythons, but I'm not against it. Maybe if it CI wasn't able to schedule all the jobs in parallel, I would feel more strongly, but I don't think that's a problem on eth-utils.

Reply:
Oh, and of course, this is what found the problem you're talking about in the first place. :man_facepalming:  So obviously worth keeping here.